conf.py

```diff
@@ -55,6 +55,17 @@
         "show off a nice portfolio in the end. Another interesting "
         "resource for web development are the MDN web docs: http://developer.mozilla.org/en-US/docs/Learn."
     ),
+    "codeblock": (
+        "Please use a codeblock when sharing code. It makes the code easier for everyone to read! "
+        "For example, you can type the following:\n\n"
+        "\\`\\`\\`\nfor(int i = 0; i < j; i++) total += total;\n\\`\\`\\`\n\n"
+        "Discord will render that as so:\n"
+        "```\nfor(int i = 0; i < j; i++) total += total;\n```\n\n"
+        "You can even get syntax highlighting by adding the language to the codeblock opening:\n\n"
+        "\\`\\`\\`cpp\nfor(int i = 0; i < j; i++) total += total;\n\\`\\`\\`\n\n"
```

Comment:
```suggestion
        "\\`\\`\\`python\nfor(int i = 0; i < j; i++) total += total;\n\\`\\`\\`\n\n"
```

(Most people who don't know how to use codeblocks probably also don't know what `cpp` is as an acronym. I suggest python or JS (and using corrosponding python or JS code)

Reply:
Changed to Python. (I don't know JS.)